scripts/release-pypath/builder/artifact_schemas.py

```diff
@@ -0,0 +1,257 @@
+    fmt = _get_index_html_template()
+
+    artifact_schema_list = "\n".join([str(artifact) for artifact in schema_data])
+
+    artifact_schema_html = textwrap.dedent(
+        f"""\
+        <ul>
+          {artifact_schema_list}
+        </ul>
+    """
+    )
+
+    return fmt.format(artifact_schema_html=artifact_schema_html)
+
+
+def _get_index_html_template() -> str:
+    return textwrap.dedent(
+        """\
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta http-equiv="content-type" content="text/html; charset=utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>dbt JSON Schema</title>
+  <style>
+body {{
+  background: #003645!important;
+  color: #93a1a1;
+  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Fira Sans","Droid Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
+  padding: 40px 20px;
+  text-align: center;
+}}
+main {{
+  font-size: 180%;
+}}
+h1 {{
+  color: #fff;
+}}
+i {{
+  color: #719e07;
+}}
+p {{
+  margin-bottom: 2em;
+}}
+a {{
+  color: #93a1a1;
+  text-decoration: none;
+}}
+a:hover {{
+  color: #fff
+}}
+ul {{
+  list-style: none;
+  padding: 0;
+
+}}
+ul li {{
+  margin-bottom: 1em;
+}}
+footer {{
+  margin-top: 10em;
+  font-size: 140%;
+}}
+  </style>
+</head>
+<body>
+  <main>
+
+    <img src="static/dbt-logo-full-white.svg" width="150">
+    <h1>dbt JSON Schema</h1>
+
+    <p>Individual JSON Schemas for dbt artifact objects</p>
+
+    {artifact_schema_html}
+  </main>
+
+  <footer>
+    <p><a href="https://github.com/fishtown-analytics/schemas.getdbt.com">github.com/fishtown-analytics/schemas.getdbt.com</a></p>
+  </footer>
+</body>
+</html>
+"""
+    )
+
+
+def check_artifact_schema(args=None):
+    env = EnvironmentInformation()
+    artifact_env = SchemaArtifactEnv(env.dbt_dir / "requirements.txt")
+    artifact_env.create(env.schemas_venv)
+    pip = str(env.schemas_venv / "bin/pip")
+    cmd = [pip, "install", "-r", "requirements.txt"]
+    stream_output(cmd, cwd=env.dbt_dir)
+
+    python_path = env.schemas_venv / "bin/python"
+    schemas_dest_dir = env.build_dir / "schemas"
+    stream_output(
+        cmd=[
+            str(python_path),
+            "scripts/collect-artifact-schema.py",
+            "--path",
+            str(schemas_dest_dir),
+        ],
+        cwd=env.dbt_dir,
+    )
+
+    artifact_schema_repo = ArtifactSchemaRepository(env.schemas_checkout_path)
+    artifact_schema_repo.clone()
+
+    deep_diff_path = str(env.schemas_venv / "bin/deep")
+    schema_files = schemas_dest_dir.glob("**/*.json")
+    for schema_file_path in schema_files:
+        with open(schema_file_path) as f:
+            relative_path = re.sub(
+                str(schemas_dest_dir) + "/", "", str(schema_file_path)
+            )
+            exists = (env.schemas_checkout_path / relative_path).exists()
+            if not exists:
+                print(f"Found new schema {relative_path}, skipping")
+                continue
```

Comment:
hmm I don't think I understand this exists check. When we don't find the path in the file system, we print that we found a new schema?